<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarbonSettingsWidget</class>
 <widget class="QWidget" name="CarbonSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QgsCollapsibleGroupBox" name="gb_ic_reference_layer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Irrecoverable carbon reference layer</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rb_local">
        <property name="toolTip">
         <string>Specify a dataset in the local computer or network drive</string>
        </property>
        <property name="text">
         <string>Local path</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rb_online">
        <property name="toolTip">
         <string>Specify the URL for the fetching the dataset from an online server</string>
        </property>
        <property name="text">
         <string>Online source</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QStackedWidget" name="sw_irrecoverable_carbon">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="pg_local_path">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QgsFileWidget" name="fw_irrecoverable_carbon"/>
          </item>
          <item>
           <widget class="QgsMapLayerComboBox" name="cbo_irrecoverable_carbon"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pg_online_source">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>URL</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="txt_ic_url">
            <property name="toolTip">
             <string>Do not include the bbox PARAM, it will be automatically appended based on the current scenario extent</string>
            </property>
            <property name="placeholderText">
             <string>Specify the URL to fetch the dataset in the CI server</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Save as</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QgsFileWidget" name="fw_save_online_file"/>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btn_ic_download">
            <property name="toolTip">
             <string>Initiate new download or refesh previous download in the background</string>
            </property>
            <property name="text">
             <string>Start download</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="SvgLabel" name="lbl_download_status_tip">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbl_ic_download_status">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="gb_stored_carbon">
     <property name="title">
      <string>Stored carbon - protect pathways</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Biomass path</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsFileWidget" name="fw_biomass"/>
      </item>
      <item row="0" column="2">
       <widget class="QgsMapLayerComboBox" name="cbo_biomass"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>178</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>SvgLabel</class>
   <extends>QLabel</extends>
   <header>cplus_plugin.gui.components.svg_label</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
